ddtrace/ext/ci.py

```diff
@@ -211,15 +216,12 @@ def extract_azure_pipelines(env):
         git.COMMIT_AUTHOR_EMAIL: env.get("BUILD_REQUESTEDFOREMAIL"),
         STAGE_NAME: env.get("SYSTEM_STAGEDISPLAYNAME"),
         JOB_NAME: env.get("SYSTEM_JOBDISPLAYNAME"),
-        # OrderedDict is necessary for comparing against a fixture in testing
```

Comment:
Kinda scratching my head at this since dictionary comparisons in Python<3.6 don't care about order, but this test failure may be related to the ordering change?

https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/39916/workflows/d8ff6592-44e6-460c-a345-e7178033348f/jobs/2680890

Reply:
Looks like it is passing now, I used `json.loads` to compare the dictionary instead. Thanks for the suggestion!